### DISCUSSION TITLE
Why does the new relese need Network Permission?

### DISCUSSION THREAD

**Discussion by Sabering1** [NickvisionApps/Denaro#696] [General]
title?

**Comment by nlogozzo** [comment]
The currency conversion tool needs to get it's data from somewhere.

**Reply by julianfairfax** [reply]
Does it fetch data in the background like GNOME Calculator does, or only when using the tool? Reason I ask is because no apps should connect to servers without the user being aware of it.

**Reply by nlogozzo** [reply]
> Does it fetch data in the background like GNOME Calculator does, or only when using the tool? Reason I ask is because no apps should connect to servers without the user being aware of it.

It uses the network when you use the tool only and the apps needs to fetch the currency conversion rates. Otherwise the network isn't touched.

**Reply by nlogozzo** [reply]
I think users would safely assume the app is connecting to the internet to get currency exchange rates...there's no other app the app could possibly know that information...
